java/src/main/java/sh/platform/languages/SampleCodeType.java

```diff
@@ -23,7 +26,10 @@
     REDIS(new RedisSample(), "Redis"),
     MEMCACHED(new MemcachedSample(), "Memcached"),
     ELASTICSEARCH(new ElasticsearchSample(), "Elasticsearch"),
-    SOLR(new SolrSample(), "Solr");
+    INFLUXDB(new InfluxdbSample(), "InfluxDB"),
```

Comment:
Until we figure out InfluxDB, maybe keep the file but don't include it in the output - same as we've done for the other languages?

Reply:
No, Java is just the first. More language will come soon, I hope. Otherwise, we remove the InfluxDB page.